cmd/minishift/cmd/start.go

```diff
@@ -416,6 +420,20 @@ func setSubscriptionManagerParameters() {
 	cluster.RegistrationParameters.Password = viper.GetString(startFlags.Password.Name)
 }
 
+func checkRegistrationCredential(host *host.Host) {
+	needRegistration, _ := registration.NeedRegistration(host)
+	if needRegistration {
+		if cluster.RegistrationParameters.Username == "" {
+			username := pkgUtil.ReadInputFromStdin("Username to register to subscription-manager")
```

Comment:
I have got this feedback in the past for vagrant-registration that  when we ask for username and password it is not inventive which username and password user supposed to enter.